harness/determined/common/experimental/oauth2_scim_client.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional
+class Oauth2ScimCient: 
+    def __init__(
+        self, 
+        id: str, 
+        domain: str, 
+        name: str,
+        secret: Optional[str] = None, 
+    ): 
+
+        self.id = id
+        self.secret = secret
+        self.domain = domain 
+        self.name = name 
```

Comment:
github indicates you don't have a newline at the end of the file.

Please fix your editor to not save files like this.  "All lines have a newline at the end" is pretty standard in unix, and if everybody does it this way then we have less code churn.